dataactvalidator/config/sqlrules/b9_award_financial.sql

```diff
@@ -7,7 +7,6 @@ SELECT af.row_number,
 	af.program_activity_code
 FROM award_financial as af
 WHERE af.submission_id = {0}
-	AND CAST(COALESCE(af.beginning_period_of_availa,'0') AS integer) >= 2016
```

Comment:
If we remove this we're not making sure we're only checking against years we have. The change that was originally here should've been kept. Why was that removed?